data/ocr/packaging.txt

```diff
@@ -30,3 +30,48 @@ Pot en plastique||pot plastique
 Pot en verre||pot verre
 Quadrette en plastique||quadrette plastique
 Sachet en plastique||sachet plastique
+Tetra Pak||tetra pak
+Tetra Pak||tetrapak
```

Comment:
What's the difference between Tetra Pak and Tetra Pack?

Reply:
no idea. spelling ?

Reply:
I think it's the same